ui/apps/platform/src/Containers/Vulnerabilities/searchOptions.ts

```diff
@@ -0,0 +1,82 @@
+import { SearchCategory } from 'services/SearchService';
+
+export type SearchOptionValue =
+    | 'SEVERITY'
+    | 'FIXABLE'
+    | 'CVE'
+    | 'IMAGE'
+    | 'DEPLOYMENT'
+    | 'NAMESPACE'
+    | 'CLUSTER'
+    | 'COMPONENT'
+    | 'COMPONENT SOURCE'
```

Comment:
Do you know if these have to be ALL CAPS? I know that's how we have it now, but if they work the same regardless if we did `COMPONENT SOURCE` or `Component Source`, it might be better to go with the second way to be consistent with the two below. But yea, not sure if that's how it works

Reply:
I think these are completely case-insensitive, but I agree we should keep them consistent. I'm neutral between ALL CAPS and sentence case, so whichever you think is better we can standardize in a follow up. (We just need to be careful with some sections, like default filters, since they rely on these values in a case-sensitive way and changes won't be caught by TypeScript at the moment.)